lti_consumer/static/js/xblock_studio_view.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Javascript for LTI Consumer Studio View.
+*/
+function LtiConsumerXBlockInitStudio(runtime, element) {
+    // Run parent function to set up studio view base JS
+    StudioEditableXBlockMixin(runtime, element);
+
+    // Define LTI 1.1 and 1.3 fields
+    const Lti1P1FieldList = [
```

Comment:
A small nitpick - I'd expect these field list constants to be camelCase instead of PascalCase.

Reply:
Fixed!